rest-api-spec/src/main/resources/rest-api-spec/test/nodes.stats/11_indices_metrics.yml

```diff
@@ -138,6 +138,35 @@
   - is_false:  nodes.$node_id.indices.translog
   - is_false:  nodes.$node_id.indices.recovery
 
+---
+"Metric - indexing doc_status":
+  - skip:
+      version: " - 2.99.99"
+      reason: "To be introduced in future release :: TODO: change if/when we backport to 2.x"
```

Comment:
Can we make changes according to ```main``` branch which 3.0.0 OS
During backport, you can change this